playerUi/separatedAudioSubtitleSettings.html

```diff
@@ -0,0 +1,303 @@
+<!DOCTYPE html>
+<!--
+*
+* Copyright (C) 2018, Bitmovin Inc, All Rights Reserved
+*
+* This source code and its use and distribution, is subject to the terms
+* and conditions of the applicable license agreement.
+*
+-->
+<html lang="en">
+<head>
+    <title>Bitmovin Demo</title>
+    <meta charset="UTF-8"/>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400" rel="stylesheet">
+    <link rel="icon" type="../image/png" href="../images/bit-fav.png">
+    <!-- Bitmovin Player -->
+    <script type="text/javascript"
+            src="//bitmovin-a.akamaihd.net/bitmovin-player/beta/7.8.2-b1/bitmovinplayer.js"></script>
```

Comment:
should we wait for a stable release or can we use the beta version here?

Reply:
Let's wait, should be out early next week anyway.